db/migrate/20230222122534_create_comments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateComments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comments do |t|
+      t.text :text
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] As requested in the comment above, the reference for the user in the comments table, should be author_id and not user_id. Kindly refer to the comment above on how to implement this.

![image](https://user-images.githubusercontent.com/86318284/220627529-571b25b1-00f9-4547-a819-7f5fe0489872.png)